src/backend/marsha/core/templates/core/lti_config.xml

```diff
@@ -18,8 +18,13 @@
     <blti:secure_launch_url>https://{{ host }}</blti:secure_launch_url>
 
     {% if icon %}
-        <blti:icon>http://{{ host }}{% static icon %}</blti:icon>
-        <blti:secure_icon>https://{{ host }}{% static icon %}</blti:secure_icon>
+        {% if environment == "development" or environment == "test" %}
+            <blti:icon>http://{{ host }}{% static icon %}</blti:icon>
+            <blti:secure_icon>https://{{ host }}{% static icon %}</blti:secure_icon>
+        {% else %}
+            <blti:icon>http:{% static icon %}</blti:icon>
+            <blti:secure_icon>https:{% static icon %}</blti:secure_icon>
+        {% endif %}
```

Comment:
In this case, I would either:

- build the full URL using Django `{{request.scheme}}://{{request.META.HTTP_HOST}}{% static icon %}`, but it's not my preferred option
- since you pass the `icon` value in the context and `blti:icon` and `blti:secure_icon` are the same, why not format it like expected directly? I see in the code, you've written a `build_absolute_uri_behind_proxy` function, may be use it if the original `request.build_absolute_uri` does not work.

Reply:
Ignore my comment, the file is on cloudfront, so this is irrelevant. The only suggestion is maybe to build the URL on the view side instead of the template side.